app/src/main/res/drawable/ic_arrow_up.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
Maybe, but each icon has a different color. So, instead of only rotate we also need to tint.